packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -561,6 +561,46 @@ describe("DropdownComponent", () => {
 				});
 			});
 
+			describe("Changing emitEvent properties ", () => {
```

Comment:
This `describe` is not really relevant since you are not "changing emitEvent properties". Actually, you are disabling/enabling the dropdown with or without the `emitEvent; false`.
Maybe you could move your 2 `it` into `describe("formControl.disabled", ()`. What do you think @nicanac ?

In that case, I would advise to you to declare the mockObserver in each `it` since it's not necessary for 
```
it("should set the right value to the mat-select when 'formControl.disabled' changes", () => {
```

Reply:
yes, why not, seems legit for me.

Reply:
moved to the `describe("formControl.disabled", ().`